modules/security-context-constraints-rbac.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+[id='role-based-access-to-ssc-{context}']
+= Role-based access to Security Context Constraints
+
+You can specify SCCs as a resource that is handled by RBAC. This allows
```

Comment:
If SCCs is plural, should it be "You can specify SCCs as resources that are handled..." or "You can specify a SCC as a resource..." ?

Reply:
Made this plural, as multiple SCCs can be defined.